Sources/Views/Buttons/AnalyticsButtonViewModel.swift

```diff
@@ -4,7 +4,7 @@ import Logging
 
 struct AnalyticsButtonViewModel: ButtonViewModel {
     let title: GDSLocalisedString
-    let icon: String?
+    let icon: ButtonIconViewModel?
```

Comment:
How come we've had to update this?

Reply:
Because we're pointing at `main` in the common repo still and didn't have [this commit](https://github.com/govuk-one-login/mobile-ios-common/commit/685092b9f29a3bc6af8629b81ffdd0ed50071808) previously